Wikia/selenium-tests
# 2 selenium tests for youtube tag in source mode case

Two tests 
CreationTicket annotation

Test Case 1:
1. Get a video id from youtube via API 
2. put tag 
<youtube> 
ID 
</youtube> to article in source mode
3. Click preview 
4. Verify that the video appears on preview

Test Case 2:
1. Get a video id from youtube via API  The ID appears in url after v"="
2. put multiline tag 
<youtube>
HXnGWKdRIOs
</youtube>  to article in source mode
